src/vaults/service/vaults/get_test.go

```diff
@@ -0,0 +1,64 @@
+package vaults
+
+import (
+	"context"
+	"testing"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+	entities2 "github.com/consensys/quorum-key-manager/src/auth/entities"
+	"github.com/consensys/quorum-key-manager/src/auth/mock"
+	"github.com/consensys/quorum-key-manager/src/entities"
+	"github.com/consensys/quorum-key-manager/src/infra/log/testutils"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetVault(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	logger := testutils.NewMockLogger(ctrl)
+	roles := mock.NewMockRoles(ctrl)
+	vault := New(roles, logger)
+
+	ctx := context.Background()
+	vaultName := "vault-id"
+	cfg := &entities.AWSConfig{}
+	allowedTenants := []string{"tenant_id_1"}
+
+	t.Run("should create and get vault client successfully", func(t *testing.T) {
+		userInfo := &entities2.UserInfo{
+			Tenant: "tenant_id_1",
```

Comment:
"tenant_id_1" and "tenant_id_2" should be consts

Reply:
done